release/128.0.yml

```diff
@@ -0,0 +1,525 @@
+release:
+  release_date: '2024-07-09'
+  text: |
+    **Thunderbird version 128.0 is only offered as direct download from
+    thunderbird.net and not as an upgrade from Thunderbird version 115 or
+    earlier. A future release will provide updates from earlier versions.**
+
+    **For more on all the new features in Thunderbird 128, see
+    [What’s New in Thunderbird 128](https://www.thunderbird.net/thunderbird/128.0/whatsnew/).**
+
+    **System Requirements:** [Details](/en-US/thunderbird/128.0/system-requirements/)
+
+    system_requirements: |
+    ## Windows
+    - Windows 10 or later
+
+    ### Windows Recommended Hardware
+    - 1GHz or faster Intel/AMD compatible processor
+    - 1GB RAM / 2GB of RAM for the 64-bit version
+    - 250 MB hard drive space
+    - additional hard drive space for mail storage
+
+    ---
+
+    ## Mac
+    - macOS 10.15 or later
+
+    ### Mac Recommended Hardware
+    - Mac computer with an Intel x86 or Apple Silicon processor
+    - 1GB RAM
+    - 500 MB hard drive space
+    - additional hard drive space for mail storage
+
+    ---
+
+    ## GNU/Linux
+    ### Software Requirements
+
+    *Please note that GNU/Linux distributors may provide packages for your distribution which have
+    different requirements.*
+
+    - Thunderbird will not run at all without the following libraries or packages:
+        - glibc 2.17 or higher
+        - GTK+ 3.14 or higher
+        - libdbus-glib 0.6.0 or higher
+        - libglib 2.42 or higher
+        - libstdc++ 4.8.1 or higher
+        - libxtst 1.2.3 or higher
+        - X.Org 1.0 or higher (1.7 or higher is recommended)
```

Comment:
These changed in 128.0, the 128.0beta.yml file has the updated Linux requirements.